strict-csp-html-webpack-plugin/README.md

```diff
@@ -73,3 +73,68 @@ You can use additional options to configure the plugin:
 | `enableTrustedTypes` | `true`  | When `true`, enables [trusted types](https://web.dev/trusted-types) for additional protections against DOM XSS attacks. |
 
 
+
+## FAQ
+
+### Does this plugin protect my users from XSS attacks?
+
+It helps, but it's not enough.
+
+To protect your site from XSS, first of all, make sure to sanitize user input. Some frameworks do this by default.
+A strict CSP is an extra security layer (also called "defense-in-depth" technique) that helps mitigate XSS attacks in case there's a sanitization bug, or other XSS bug, in your application or dependency. A strict CSP prevents the execution of malicious scripts in case they make it to your application.
+
+### Where should I use this plugin?
+
+This plugin is best-suited for single-page applications. 
+If you have server-side logics, use a [nonce-based strict CSP](https://web.dev/strict-csp/#step-1:-decide-if-you-need-a-nonce-or-hash-based-csp) instead.
+
+### How does this plugin differ from [csp-html-webpack-plugin](https://www.npmjs.com/package/csp-html-webpack-plugin)?
+
+This plugin focuses on one thing: it sets up a [strict CSP](https://web.dev/strict-csp), that is, an efficient defense-in-depth mechanism against XSS attacks.
+
+csp-html-webpack-plugin is more flexible. If you're using a CSP for other purposes than XSS mitigation, check out [csp-html-webpack-plugin](https://www.npmjs.com/package/csp-html-webpack-plugin).
```

Comment:
Sounds good, used your suggestion, with a tweak (_Note that at the moment, static nonces risk making csp-html-webpack-plugin's CSP bypassable, though this may be resolved in the future._).